docs/event-bus/docs/001-overview-event-bus.md

```diff
@@ -4,6 +4,6 @@ title: Overview
 
 Kyma Event Bus enables the integration of various external solutions with Kyma. The integration is achieved using the `publish-subscribe` messaging pattern that allows Kyma to receive business Events from different solutions, enrich them, and trigger business flows using lambdas or services defined in Kyma.
 
-To learn how to write an HTTP service or Lambda in Kyma, and handle the Event Bus published Events, check the Services Programming Model [guide](013-details-service-programming-model.md) and the Lambda Programming Model [guide](../../serverless/docs/035-programming-model.md).
+To learn how to write an HTTP service or lambda in Kyma, and handle the Event Bus published Events, check the **Services Programming Model** [guide](013-details-service-programming-model.md) and the **Lambda Programming Model** [guide](../../serverless/docs/035-programming-model.md).
```

Comment:
remove cross-document links please

Reply:
It's not the scope of my task, but sure, I can do this.

Reply:
Ah :O